test/CMakeLists.txt

```diff
@@ -218,6 +220,7 @@ if( iodaconv_bufr_ENABLED )
     testoutput/bufr_read_wmo_radiosonde.nc
     testoutput/NC005031.nc
     testoutput/NC005066.nc
+    testoutput/Out_NC005080.nc
```

Comment:
Reflect the new filename here, i.e.
testinput/NC005080.nc will become testinput/bufr_ncep_satwind_avhrr.nc

Reply:
I've updated CMakeLists to match the above. Will be re-uploaded shortly.